scripts/ci/pack_soljson.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+set -euo pipefail
+
+SCRIPT_DIR="$(realpath "$(dirname "$0")")"
+SOLJSON_JS="$1"
+SOLJSON_WASM="$2"
+SOLJSON_WASM_SIZE=$(wc -c "${SOLJSON_WASM}" | cut -d ' ' -f 1)
+OUTPUT="$3"
+
+(( $# == 3 )) || { >&2 echo "Usage: $0 soljson.js soljson.wasm packed_soljson.js"; exit 1; }
+
+# If this changes in an emscripten update, it's probably nothing to worry about,
+# but we should double-check when it happens and adjust the tail command below.
+[ "$(head -c 5 "${SOLJSON_JS}")" == "null;" ] || { >&2 echo 'Expected soljson.js to start with "null;"'; exit 1; }
+
+echo "Packing $SOLJSON_JS and $SOLJSON_WASM to $OUTPUT."
+(
+	echo -n 'var Module = Module || {}; Module["wasmBinary"] = '
+	echo -n '(function(source, uncompressedSize) {'
+	# Note that base64DecToArr assumes no trailing equals signs.
+	cat "${SCRIPT_DIR}/base64DecToArr.js"
+	# Note that mini-lz4.js assumes no file header and no frame crc checksums.
+	cat "${SCRIPT_DIR}/mini-lz4.js"
```

Comment:
Good thing that these are not minified - we'll finally get some sensible stack traces in corner cases that don't include the whole binary :) While at it, maybe you could also disable minification for the part injected by emscripten?

Reply:
I'd rather keep more changes separate. At least separate from this PR, but preferable even in a separate release :-).

Reply:
I think a separate PR is fine but why a separate release? Are you afraid that it will somehow interfere with packing?

Reply:
*If* something breaks for anyone anywhere, we at least have one likely cause, resp. can track a likely cause down by having people try specific releases :-). Just beautifying soljson.js would probably be fine, but changing the optimization settings in emscripten by adding ``-g2`` or ``--profiling`` may have arbitrary effects from all I know :-).
But yeah, I'm not entirely serious, we can look into it in a separate PR.

Reply:
By the way: do we *really* need this stuff to be un-minified? Or is it rather that the encoded binary is annoying because it ends up in error outputs? Because if I just adjust the line-breaks when stitching the files together slightly, it'd get its own line...

Reply:
It's the latter but I just don't see much benefit in minifying it.

Reply:
Slightly adjusted the line-breaks for this in the last commit anyways, i.e. it puts the binary in a single ``"...",`` line.